ses-client/src/main/java/com/amazonaws/client/AmazonSESClient.java

```diff
@@ -0,0 +1,77 @@
+import com.amazonaws.services.simpleemail.model.Message;
+import com.amazonaws.services.simpleemail.model.SendEmailRequest;
+
+import java.io.IOException;
+
+public class AmazonSESClient {
+
+    // Replace sender@example.com with your "From" address.
+    // This address must be verified with Amazon SES.
+    static final String FROM = "sender@example.com";
+
+    // Replace recipient@example.com with a "To" address. If your account
+    // is still in the sandbox, this address must be verified.
+    static final String TO = "recipient@example.com";
+
+    // The configuration set to use for this email. If you do not want to use a
+    // configuration set, comment the following variable and the
+    // .withConfigurationSetName(CONFIGSET); argument below.
+    static final String CONFIGSET = "ConfigSet";
+
+    // The subject line for the email.
+    static final String SUBJECT = "Amazon SES test (AWS SDK for Java)";
+
+    // The HTML body for the email.
+    static final String HTMLBODY = "<h1>Amazon SES test (AWS SDK for Java)</h1>"
+            + "<p>This email was sent with <a href='https://aws.amazon.com/ses/'>"
+            + "Amazon SES</a> using the <a href='https://aws.amazon.com/sdk-for-java/'>"
+            + "AWS SDK for Java</a>";
+
+    // The email body for recipients with non-HTML email clients.
+    static final String TEXTBODY = "This email was sent through Amazon SES "
+            + "using the AWS SDK for Java.";
+
+    public static void main(String[] args) throws IOException {
+        try {
+            AwsClientBuilder.EndpointConfiguration endpointConfiguration =
+                    new AwsClientBuilder.EndpointConfiguration("http://localhost:8080", "eu-west-2");
+
+            AmazonSimpleEmailService client = AmazonSimpleEmailServiceClientBuilder.standard()
+                    .withEndpointConfiguration(endpointConfiguration)
+                    .withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials("x", "x")))
+                    .build();
+            
+            SendEmailRequest request = new SendEmailRequest()
+                    .withDestination(
+                            new Destination().withToAddresses(TO))
+                    .withMessage(new Message()
+                            .withBody(new Body()
+                                    .withHtml(new Content()
+                                            .withCharset("UTF-8").withData(HTMLBODY))
+                                    .withText(new Content()
+                                            .withCharset("UTF-8").withData(TEXTBODY)))
+                            .withSubject(new Content()
+                                    .withCharset("UTF-8").withData(SUBJECT)))
+                    .withSource(FROM)
+                    // Comment or remove the next line if you are not using a
+                    // configuration set
+                    .withConfigurationSetName(CONFIGSET);
```

Comment:
The Real SES fails at this point. We are not using ConfigSets but we should simulate this in the local-ses. Should reject any send request that has this field set. 
After we reproduce the failure locally we should remove the ConfigSet line.

Reply:
Will apply this in the other project.